ils-A-analog.py

```diff
@@ -0,0 +1,26 @@
+from irods.test.helpers import make_session
```

Comment:
I think this will be removed from the final PR, as it was only useful for establishing the erroneous behavior.

Reply:
@trel   `acls` is shorter and can eventually replace `permissions`.  I did not want to change the value of a default parameter in a way that would likely break a lot of existing code.

Reply:
okay, please remove this file and make a proper test that exercises the new code.

Reply:
The test now included asserts the conditions looked for (inclusion of only the group ACLs, because they're catalog objects; and exclusion of group members for whom catalog ACLs do not exist).